configs/gtp-base.cfg

```diff
@@ -0,0 +1,26 @@
+@include compute/1gpu.cfg
+
+# We're not really interested in detailed logging when running with GTP
+logSearchInfo = false
+logAllGTPCommunication = false
+
+# For compatibility with other engines, we need to set multiStoneSuicideLegal, which
```

Comment:
is this referring to the ELF/Leela transfer experiments? When we're using GTP to run KataGo vs. ELF we do want multiStoneSuicideLegal = false because ELF rejects suicide moves. But if we're using GTP to play against our adversaries on Sabaki, we probably want to leave multiStoneSuicideLegal = true since that's how our adversaries were trained

Reply:
Do we need to have a separate config file for that? Also now that you mention it we might want to actually train our adversaries with `multiStoneSuicideLegals=true,false`...

Reply:
I think when writing my comment I was just thinking that the way we had it before with `multiStoneSuicideLegal = true` in the config and with `compose/transfer/compose.yml` overriding it to false made sense to me — by default we have `multiStoneSuicideLegal = true` since Tromp-Taylor allows suicide moves0, but we're forced to turn it off specifically for transfer experiments due to ELF's constraints. But I don't really have strong feelings about how we handle this

Reply:
Having the KataGo vs KataGo evaluation be "on-distribution" by sticking to Tromp-Taylor seems important. But if we do that exclusively using match (just using KataGo vs KataGo GTP as a sanity check) then it might be OK to default to true in GTP for maximum compatibility with other engines.

Reply:
You mean it might be OK to default to false?

Reply:
Yep, sorry, I meant default to false seems OK.